services/navigation/builtin/builtin.go

```diff
@@ -216,11 +213,11 @@ type builtIn struct {
 	mode      navigation.Mode
 	mapType   navigation.MapType
 
-	base              base.Base
-	movementSensor    movementsensor.MovementSensor
-	obstacleDetectors []*ObstacleDetector
-	motionService     motion.Service
-	obstacles         []*spatialmath.GeoObstacle
+	base                 base.Base
+	movementSensor       movementsensor.MovementSensor
+	motionService        motion.Service
+	exploreMotionService motion.Service
```

Comment:
[note] This again will be going away after merge, should I call this out specifically here?